js/kraken.js

```diff
@@ -971,23 +971,7 @@ module.exports = class kraken extends Exchange {
         }
         return {
             'id': id,
-            'clientOrderId': clientOrderId,
             'info': response,
-            'timestamp': undefined,
-            'datetime': undefined,
-            'lastTradeTimestamp': undefined,
-            'symbol': symbol,
-            'type': type,
-            'side': side,
```

Comment:
The same things are true for `type` and `side` – both are fine and don't change over time.